run.sh

```diff
@@ -1,10 +1,10 @@
 # Test credentials to build the EKS cluster in our lab environment
 # 
 
-export AWS_ACCESS_KEY_ID="AKIARRPLFNP7IU6BFQ5U"
-export AWS_SECRET_ACCESS_KEY="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"
-export AWS_DEFAULT_REGION="ap-southeast-1"
+export AWS_ACCESS_KEY_ID="<AWS ACCESS KEY>"
+export AWS_SECRET_ACCESS_KEY="<AWS SECRET ACCESS KEY>"
+export AWS_DEFAULT_REGION="<REGION>"
 
 terraform apply -var-file=test.tfvars
-#pause
-#terraform destroy -var-file=test.tfvars
+pause
```

Comment:
what does pause do?

Reply:
Nothing, on linux. I think my brain was in DOS land when I wrote that. I'll remove it. Thanks for picking up on it!